Rotativa.NetCore/Rotativa.NetCore.csproj

```diff
@@ -28,27 +28,26 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.Cookies" Version="1.0.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
   </ItemGroup>
 
   <PropertyGroup>
     <CodeAnalysisRuleSet>..\stylecop.ruleset</CodeAnalysisRuleSet>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-    <Description>A port of the Rotativa library for the ASP.NET Core library.</Description>
-    <PackageReleaseNotes>This is the initial version of the port and might have some bugs as it is not fully tested.</PackageReleaseNotes>
+    <Description>A port of the Rotativa library for the ASP.NET Core library, compatible with .NET Core 2.0.</Description>
+    <PackageReleaseNotes>Porting for .NET Core 2.0</PackageReleaseNotes>
     <Copyright>Copyright 2017</Copyright>
-    <Authors>Alexandru Munteanu</Authors>
-    <Company />
+    <Authors>Stefano Baruzzo</Authors>
```

Comment:
At least have the decency to append your name to the original author if you only changed the package version. Also you have a conflict. Please resolve that.